content/release/8.3.md

```diff
@@ -0,0 +1,241 @@
+---
+Title: FRRouting Release 8.3
+Release: 8.3
+GitHub: https://github.com/FRRouting/frr/releases/tag/frr-8.3
+Date: 2022-07-18
+---
+
+FRRouting Release 8.3 Available for Download
+==============================================
+
+*July 18, 2022*
+
+We are pleased to announce FRR 8.3.
+
+FRR 8.3 brings a long list of enhancements and fixes with around 1034 commits from 67 developers. Thanks to all contributors.
+
+Debian Packages - https://deb.frrouting.org
+
+RPM Packages - https://rpm.frrouting.org
+
+Snaps - https://snapcraft.io/frr
+
+Docker - [frrouting/frr:v8.3](https://hub.docker.com/layers/frrouting/frr/v8.3/images/sha256-b236f9d9d6f8bffa54bf0cb794baf5ad0724bdd8e809b68d2715f8705d84e7ad)
```

Comment:
Should be `https://hub.docker.com/layers/frr/frrouting/frr/v8.3.0/images/sha256-2335df29c48d7fb4a12c8830b2f5e5fcab6562799ccc82d1b09aadc964eb9dd5`